src/main.js

```diff
@@ -1 +1,53 @@
-console.log('Hello, world!');
+// something cool i learned :
+// before i chose mousedown n click as an event so if i click on the value the chosen input changes but the function doesnt work because the event is mouse down it must exactly be click on the radio button..... 
+
+const form = document.forms[0];
+const payment = form.elements.plan;
+
+payment.forEach( function (pembayaran){ 
+  pembayaran.addEventListener('change',function(){
+     if (form.plan.value === 'Monthly') {
+    document.querySelector('.form__total').textContent = '30$';
+    document.querySelector('.form__pay span').textContent = '30$';};
+
+    if (form.plan.value === 'Annually') {
+    document.querySelector('.form__total').textContent = '300$';
+    document.querySelector('.form__pay span').textContent = '300$';}
+
+  });
+});
+
+
+// enable button !!!!!!
+const payButton = form.elements.submit
+const money = document.querySelector('.form__pay span');
+ if (money.textContent !='$0') {
+   payButton.disabled = false;
+ }
+
+// payPAl Amount Bug
+//  need help in this :(((
+
+let payPal = document.querySelector     ('.ppaf-input');
+ payPal.setAttribute('value', money.textContent)
```

Comment:
You'll need this for the paypal bug:

```javascript
const paypalLink = document.querySelector('a.form__cc-heading');
paypalLink.setAttribute('href', `https://paypal.me/nikosolihin/${amount}`);
```

The amount is the same one you set with `.textContent` above, but without the dollar sign — so it has to alternate between 30 and 300.

Might also be better to put this whole paypal logic within your `pembayaran.addEventListener` conditional logic.